src/listconf.c

```diff
@@ -63,6 +67,8 @@ extern void cuda_device_list();
 #endif
 #if HAVE_OPENCL
 #include "common-opencl.h"
+#define _BSD_SOURCE 1
+#define _DEFAULT_SOURCE 1
```

Comment:
Feature macros are supposed to be defined prior to including *any* system header so these should be moved up above the `#define NEED_OS_FLOCK`.